apps/server/src/setup.ts

```diff
@@ -80,11 +80,26 @@ export const pathToStartDb = isTest
   ? join(currentDirectory, '../', config.database.testdb, config.database.filename)
   : join(currentDirectory, '/preloaded-db/', config.database.filename);
 
+//TODO: move all to external
```

Comment:
suggestion: can you clarify this comment for future us? what is all?
Would the note be TO`DO: move all static files to the external directory?`